packages/interaction/src/InteractionManager.ts

```diff
@@ -1050,6 +1050,10 @@ export class InteractionManager extends EventEmitter
     public setCursorMode(mode: string): void
     {
         mode = mode || 'default';
+        if (!this.interactionDOMElement.style)
```

Comment:
Would be helpful to add a comment here about why you're bailing without a style prop

Reply:
offscreen canvas does not support setting style, changed code to better reflect this logic